arrays/Kadane's Algorithm.md

```diff
@@ -0,0 +1,51 @@
+---
+id: kadanes-algorithm
+title: Kadane's Algorithm
+sidebar_label: Introduction to Kadane's Algorithm
+sidebar_position: 2
+description: Kadane's Algorithm efficiently finds the maximum sum of a contiguous subarray. It is a fundamental algorithm for solving problems involving subarray sums.
```

Comment:
replace `description: Kadane's Algorithm efficiently finds the maximum sum of a contiguous subarray. It is a fundamental algorithm for solving problems involving subarray sums.` with `description: "Kadane's Algorithm efficiently finds the maximum sum of a contiguous subarray. It is a fundamental algorithm for solving problems involving subarray sums."`